src/store/getters.ts

```diff
@@ -0,0 +1,50 @@
+import { ContextState, MolgenisMenu, MolgenisFooter } from '../types'
+
+export const molgenisMenu = (state: ContextState): MolgenisMenu | null => {
+  const context = state.context
+  if (!context || !context.menu) {
+    return null
+  }
+  const result: MolgenisMenu = {
+    menu: context.menu,
+    helpLink: context.helpLink,
+    authenticated: context.authenticated,
+    selectedPlugin: state.selectedPlugin,
+    loginHref: context.loginHref
+  }
+  // map optional keys
+  if (context.logoTop) {
+    result.topLogo = context.logoTop
+  }
+  if (context.navBarLogo) {
+    result.navBarLogo = context.navBarLogo
+  }
+  if (context.logoTopMaxHeight) {
```

Comment:
the number 0 is a falsy value, it should still be set

Reply:
updated with hasOwnProperty check